website/pages/en/index.js

```diff
@@ -84,54 +74,247 @@ class Index extends React.Component {
+                }, 
+                {
+                  title: 'Optimised for UI',
+                  content: '<p>The mutation API makes it trivial to do optimistic rendering, error handling and reverting when things don’t go as planned</p>'
+                }
               ]}
+            />
+          </Container>
+
+          <Container className='textSection aheadSection'>
+            <h2>Ahead-of-time Safety</h2>
+            <GridBlock
               layout="threeColumn"
+              contents={[
+                {
+                  title: 'Peace of Mind',
+                  content: '<p>While you work on a Relay project, the Relay compiler will guide you to ensure project-wide consistency.</p>'
+                },
+                {
+                  title: 'Runtime Optimised',
+                  content: '<p>Relay pre-computes runtime lookups on the developer’s computer, not your users.</p>'
+                }, 
+                {
+                  title: 'Isolated Interfaces',
+                  content: '<p>Each React component using Relay gets unique Flow or TypeScript interfaces generated.</p>'
+                }
+              ]}
             />
           </Container>
 
-          <Container padding={['bottom', 'top']} background="light">
+          <Container className='textSection relaySection'>
+            <h2>Can Relay Work For Me?</h2>
             <GridBlock
+              layout="twoColumn"
               contents={[
                 {
-                  content:
-                    'Relay Modern is a new version of Relay designed from the ground up to be easier to use, more extensible and, most of all, able to improve performance on mobile devices. Relay Modern accomplishes this with static queries and ahead-of-time code generation. Incrementally convert existing Relay apps, or start a new one with Relay Modern.',
-                  imageAlign: 'center',
-                  image: '',
-                  title: 'Relay Modern',
+                  title: 'Adopt Incrementally',
+                  content: '<p>If you already can render React components, you’re most of the way there. Relay requires a Babel plugin, and to also run the Relay Compiler.</p><p>You can use Relay out of the box with Create React App and Next.js.</p>'
                 },
+                {
+                  title: 'Make Complexity Explicit',
+                  content: '<p>Relay requires a bit more up-front setup and tools, in favour of supporting an architecture of isolated components which <a href="#">can scale</a> with your team and app complexity.</p>'
+                }, 
+                {
+                  title: 'User Interface Platform',
+                  content: '<p>Relay strives to offer a set of opinionated primtive React components, on which you can build any type of data-driven application.</p><p>Learn these principals once, then your projects spend more time working on business logic instead of pipelining data.</p>'
```

Comment:
Typos:
- primtive (missing i)
- principals (should be principles)

> Learn these principals once, then your projects spend more time working on business logic instead of pipelining data.

I get the meaning, but this sentence sounds awkward. Maybe drop "your projects".